src/upload.ts

```diff
@@ -220,7 +220,7 @@ export async function uploadLargeFileRequest(
 
   // TODO: Add back upload options once they are implemented in skyd.
   const url = await buildRequestUrl(this, { endpointPath: opts.endpointLargeUpload });
-  const headers = buildRequestHeaders(undefined, opts.customUserAgent, opts.customCookie);
+  const headers = buildRequestHeaders(undefined, opts.customUserAgent, opts.customCookie, opts.APIKeyPortal);
```

Comment:
can we rename the option to `SkynetApiKey` to better resemble the actual header name ?

Reply:
That would be great! It is confusing with the other `APIKey` option, which is actually the API password for `skyd`. It might be nice to rename that one as well, as there's a difference between a password and an API key.

Reply:
Unfortunately, we can't rename that option as it's part of the public API, but we can deprecate it and add a new option.

Reply:
https://github.com/SkynetLabs/skynet-js/issues/394

Reply:
Oh, right, we talked about that. It's nice to have it in a F/U but definitely not a top priority.